src/main/java/org/mule/extension/sftp/internal/connection/SftpConnectionSettings.java

```diff
@@ -73,6 +76,14 @@
   @Path(type = FILE)
   private String identityFile;
 
+
+  @Parameter
+  @Optional(defaultValue = "NativePRNG")
```

Comment:
Is it available on Windows?

Reply:
https://github.com/frohoff/jdk8u-dev-jdk/blob/master/src/windows/classes/sun/security/provider/NativePRNG.java

Reply:
I've updated this so that the default configuration PRNG is AUTOSELECT. In MulePRNGAlgorithm, in case Autoselect is selected (and accordingly there is no algorithmName specified) a SecureRandom is generated without an algorithm so it will select it depending on the OS behind. In case a PRNG algorithm which is not supported by the OS is selected, it will also fallback to SecureRandom without parameters like in JSCH.